build.ps1

```diff
@@ -53,10 +53,9 @@ Param(
     [Alias("DryRun","Noop")]
     [switch]$WhatIf,
     [switch]$Mono,
-    [switch]$SkipToolPackageRestore
-    # ,
-    # [Parameter(Position=0,Mandatory=$false,ValueFromRemainingArguments=$true)]
-    # [string[]]$ScriptArgs
+    [switch]$SkipToolPackageRestore,
+    [string]$SigningCertificatePath = "./certificates/OctopusDevelopment.pfx",
+    [string]$SigningCertificatePassword = "Password01!"

```

Comment:
These are only used for development, do you still think we should not have them ?